features/fout-meldingen.feature

```diff
@@ -0,0 +1,220 @@
+
+        Voorbeelden:
+        | path                   | query string                                                                                      | fout detail                                                                                                |
+        | /adressen              | ?pandIdentificatie=0226100000008856&adresseerbaarObjectIdentificatie=0226010000038820             | Alleen parameter 'pandIdentificatie' of 'adresseerbaarObjectIdentificatie' moet zijn opgegeven             |
+        | /adresseerbareobjecten | ?nummeraanduidingIdentificatie=0226200000038923&pandIdentificatie=0226100000008856                | Alleen parameter 'nummeraanduidingIdentificatie' of 'pandIdentificatie' moet zijn opgegeven                |
+        | /panden                | ?adresseerbaarObjectIdentificatie=0226010000038820&nummeraanduidingIdentificatie=0226200000038923 | Alleen parameter 'adresseerbaarObjectIdentificatie' of 'nummeraanduidingIdentificatie' moet zijn opgegeven |
+
+Rule: alle parameter fouten in een request worden samen geretourneerd
+
+    Scenario: er zijn meerdere verschillende fout soorten
+        Als '/panden?adresseerbaarObjectIdentificatie=0226010000038820&nummeraanduidingIdentificatie=0226200000038923&fields=bestaatniet&expand=bestaatookniet' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                                                                                                                                                |
+        | title  | Een of meerdere parameters zijn niet correct.                                                                                                                                         |
+        | status | 400                                                                                                                                                                                   |
+        | detail | Alleen parameter 'adresseerbaarObjectIdentificatie' of 'nummeraanduidingIdentificatie' moet zijn opgegeven.<br />Parameters 'fields', 'expand' bevatten een (deels) ongeldige waarde. |
+        En bevat de response de volgende invalidParams
+        | name   | reason                                       |
+        | fields | ongeldige waarde: 'bestaatniet' opgegeven    |
+        | expand | ongeldige waarde: 'bestaatookniet' opgegeven |
+
+Abstract Scenario: opgegeven identificatie is niet gekoppeld aan een resource
+    Als '<path>/<identificatie>' wordt aangeroepen
+    Dan bevat de response de volgende kenmerken
+    | naam   | waarde                                                            |
+    | title  | Opgevraagde <resource naam> bestaat niet.                         |
+    | status | 404                                                               |
+    | detail | Geen <resource naam? gevonden met identificatie '<identificatie>' |
```

Comment:
Corrigeer in

| detail | Geen <resource naam> gevonden met identificatie '<identificatie>' |